src/components/screen.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable react/prefer-stateless-function */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class Screen extends Component {
+  render() {
+    // console.log(this.props.state.total);
+    // const {props} = this;
+    // todo: line 9 and 10 works
+    // const { ...data } = this.props;
+    // console.log(data.state.total);
+    // const { data } = this.props;
+    // console.log(data);
```

Comment:
- [ ] Please remove  the commented out code

Reply:
I have done that. Thanks